src/renderer/components/Experiment/Diffusion/Diffusion.tsx

```diff
@@ -844,6 +853,56 @@ export default function Diffusion({ experimentInfo }: DiffusionProps) {
                           sx={{ width: 100 }}
                         />
                       </FormControl>
+                      {inputImageBase64 && (
+                        <Stack direction="row" spacing={2} alignItems="center">
+                          <FormControl sx={{ minWidth: 160 }}>
+                            <LabelWithTooltip tooltip="Manage and select a ControlNet to guide image generation.">
+                              ControlNet
+                            </LabelWithTooltip>
+                            <Button
+                              size="sm"
+                              variant="outlined"
+                              onClick={() => setControlnetModalOpen(true)}
+                            >
+                              {controlNetType === 'off'
+                                ? 'Select ControlNet'
+                                : controlNetType}
+                            </Button>
+                          </FormControl>
+
+                          {controlNetType !== 'off' && (
+                            <FormControl sx={{ minWidth: 160 }}>
+                              <LabelWithTooltip tooltip="Select how to process the reference image for the ControlNet model.">
+                                Process Type
+                              </LabelWithTooltip>
+                              <select
```

Comment:
Remember to use <Select> from MUI Joy (https://mui.com/joy-ui/react-select/), not <select> which is a default HTML component.